modules/helper-functions.js

```diff
@@ -12,7 +20,115 @@ const countItemsOccurency = items =>
     return acc;
   }, {});
 
+const setCalendar = (names, counts, days) => {
+  const calendar = [];
+
+  for (let i = days; i >= 0; i--) {
+    const today = Date.now();
```

Comment:
Why you call `Date.now()` in cycle when you can call it out of `for (...)` scope?

Reply:
Good point. 
Thanks
solved